sklearn/metrics/_classification.py

```diff
@@ -1787,7 +1787,7 @@ def balanced_accuracy_score(y_true, y_pred, *, sample_weight=None,
     deal with imbalanced datasets. It is defined as the average of recall
     obtained on each class.
 
-    The best value is 1 and the worst value is 0 when ``adjusted=False``.
```

Comment:
This change seems unrelated.

Reply:
Sorry, I accidentally started off the wrong branch, I'll correct this now